components/ui/timeline.tsx

```diff
@@ -0,0 +1,42 @@
+type TimelineProps = {
+  active: string;
+};
+
+export function Timeline({ active }: TimelineProps) {
+  return (
+    <div className="flex flex-row items-center justify-center mt-2">
+      <p
```

Comment:
Out of curiosity, why make this a `<p>` rather than a `<div>`, which seems semantically more appropriate?

Reply:
Ah, it just totally blanked my mind that I can add text to `<div>`! I agree that `<div>` seems more appropriate here

Reply:
Great, thanks!